unclognearline/src/main/scala/UnclogNearline.scala

```diff
@@ -43,7 +43,9 @@ object UnclogNearline extends ZonedDateTimeEncoder with VSFileStateEncoder with
     sys.env.getOrElse("ES_PORT","9200").toInt
   )
 
-  lazy val storageId = sys.env.get("STORAGE_IDENTIFIER")
+  lazy val storageId = sys.env.get("STORAGE_IDENTIFIER") match {
+    case Some(id)=>id
```

Comment:
you'll need to put a None case in here too, otherwise you will get an exception when the match is evaluated.
personally I would remove the `lazy` qualifier so it's evaluated up-front, and put a None case that prints an error message and exists using `complete_run()`. (Aside: if you have a running ActorSystem, then an exception in the main thread does not terminate the program and it gets stuck in a zombie state)

Reply:
Done.